tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -57,7 +57,24 @@ def running_vm_for_migration(
 
 
 @pytest.fixture()
-def migrated_vmi(running_vm_for_migration):
+def inter_vm_connectivity_exists(
+    running_vm_static,
+    running_vm_for_migration,
+):
+    static_vm_ip = running_vm_static.vmi.interfaces[0]["ipAddress"]
+
+    vm_console_run_commands(
```

Comment:
i'd expect this to be as part of `running_vm_for_migration` - the fixture should yield a properly configured VM and connectivity is one such thing.

Reply:
I'm changing it, but I'd like to further discuses it.
To me it seems like connection two separate actions into a single function instead of doing a single action in each fixture. This also introduce dependency in the VM creation - the destination VM must be created before the target VM. Can you please further explain why these two actions should be handled in a single fixture in this case?